File: javascript/resistor-color-duo/resistor-color-duo.js
const BANDS  = ['black', 'brown', 'red', 'orange', 'yellow', 'green', 'blue', 'violet', 'grey', 'white'];

export const decodedValue = (colors) => {
  return BANDS.indexOf(colors[0]) * 10 + BANDS.indexOf(colors[1]);
}

File: ruby/resistor-color-duo/resistor_color_duo.rb
class ResistorColorDuo
  def self.value(colors)
    bands = ["black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"]

    bands.index(colors[0]) * 10 + bands.index(colors[1])
  end
end
